### DISCUSSION TITLE
How display reCaptcha?

### DISCUSSION THREAD

**Discussion by Diman96** [OrchardCMS/OrchardCore#6875] [Q&A]
Did anyone work with captcha? how to display it on a custom liquid page so that it looks similar to ` {% antiforgerytoken %} ` in code (workflow and reCaptcha are already configure)?

**Comment by Piedone** [comment]
While I haven't done this yet, I think the `<recaptcha>` tag helper that's used e.g. [here](https://github.com/OrchardCMS/OrchardCore/blob/6cddd9f9904ff117ba82587f39fc76f49fac3cc3/src/OrchardCore.Modules/OrchardCore.ReCaptcha/Views/Items/ReCaptchaPart.cshtml#L5) should be accessible form Liquid too, [as all tag helpers are](https://docs.orchardcore.net/en/dev/docs/reference/modules/Liquid/#tag-helper-tags).

Do note that this is only to display the captcha, you'll also need to validate it when handling the post, [as explained in the docs](https://docs.orchardcore.net/en/dev/docs/reference/modules/ReCaptcha/).

**Reply by Diman96** [reply]
unfortunately it doesn't work on liquid, https://github.com/OrchardCMS/OrchardCore/issues/2768.

**Reply by Piedone** [reply]
Oh, well. Then it just requires a Liquid registration. Would you like to venture into contributing it? You can check out what's done with other TagHelper classes for inspiration.

**Reply by Diman96** [reply]
Could you give me a link to a code that does what you have mentioned?

**Reply by Piedone** [reply]
Wait, I mixed it up with another Liquid thing. You don't actually need to explicitly register it. Let me continue under the issue.
